### ISSUE TITLE
White/Blue screen

### ISSUE THREAD

**Issue by JHughes89** [Spectre0987/TardisMod#299]
whenever I change my interior using any tardismod custom blocks or any blocks from another mod, the screen goes blue/white when I step outside the tardis and crashes. This also happened when went to the void but the screen only went white/blue when I looked at the door but when i faced the console the light disappeared  - so I had to use the door control on the console to shut the door.

**Comment by Suffril** [comment]
Screenshots? Crash report?

**Comment by JHughes89** [comment]
---- Minecraft Crash Report ----

WARNING: coremods are present:
  Inventory Tweaks Coremod (InventoryTweaks-1.63.jar)
  CTMCorePlugin (CTM-MC1.12.2-0.3.3.22.jar)
Contact their authors BEFORE contacting forge

// I feel sad now :(

Time: 5/23/19 7:08 PM
Description: Rendering screen

java.lang.NullPointerException: Rendering screen
	at net.minecraft.client.gui.inventory.GuiContainer.func_146977_a(GuiContainer.java:243)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:98)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:396)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1124)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1119)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.minecraft.client.gui.inventory.GuiContainer.func_146977_a(GuiContainer.java:243)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:98)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:396)

-- Screen render details --
Details:
	Screen name: com.swdteam.client.gui.GuiOxygenTankRefiller
	Mouse location: Scaled: (221, 121). Absolute: (884, 484)
	Screen size: Scaled: (442, 243). Absolute: (1768, 969). Scale factor of 4

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['JHughes89'/312, l='MpServer', x=75.34, y=59.50, z=198.96]]
	Chunk stats: MultiplayerChunkCache: 441, 441
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (28,64,140), Chunk: (at 12,4,12 in 1,8; contains blocks 16,0,128 to 31,255,143), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 1201431 game time, 135435 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: true), thunder time: 0 (now: true)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 4 total; [EntityArmorStand['Armor Stand'/4400, l='MpServer', x=77.50, y=59.00, z=214.50], EntityPlayerSP['JHughes89'/312, l='MpServer', x=75.34, y=59.50, z=198.96], EntityOtherPlayerMP['SolarStream72'/337, l='MpServer', x=203.41, y=60.42, z=211.70], EntityDavrosChair['Davros Chair'/4399, l='MpServer', x=77.00, y=59.00, z=194.93]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:420)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2741)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:419)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 8 (amd64) version 6.2
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 231952464 bytes (221 MB) / 2147483648 bytes (2048 MB) up to 2147483648 bytes (2048 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP 9.42 Powered by Forge 14.23.5.2838 29 mods loaded, 29 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State  | ID              | Version                  | Source                                        | Signature                                |
	|:------ |:--------------- |:------------------------ |:--------------------------------------------- |:---------------------------------------- |
	| LCHIJA | minecraft       | 1.12.2                   | minecraft.jar                                 | None                                     |
	| LCHIJA | mcp             | 9.42                     | minecraft.jar                                 | None                                     |
	| LCHIJA | FML             | 8.0.99.99                | forge-1.12.2-14.23.5.2838.jar                 | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| LCHIJA | forge           | 14.23.5.2838             | forge-1.12.2-14.23.5.2838.jar                 | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| LCHIJA | mcmultipart     | 2.5.0                    | MCMultiPart-Mod-1.12.2.jar                    | None                                     |
	| LCHIJA | sonarcore       | 5.0.9                    | Sonar-Core-1.12.2.jar                         | None                                     |
	| LCHIJA | bagelsmore      | 2.0.0                    | Bagelsmore-Mod-1.12.2 (1).jar                 | None                                     |
	| LCHIJA | biomesoplenty   | 7.0.1.2419               | BiomesOPlenty-1.12.2-7.0.1.2419-universal.jar | None                                     |
	| LCHIJA | blockcraftery   | 1.12.2-1.1.0             | blockcraftery-1.12.2-1.1.0.jar                | None                                     |
	| LCHIJA | cfm             | 6.2.1                    | cfm-6.2.1-mc1.12.2.jar                        | None                                     |
	| LCHIJA | ctm             | MC1.12.2-0.3.3.22        | CTM-MC1.12.2-0.3.3.22.jar                     | None                                     |
	| LCHIJA | chisel          | MC1.12.2-0.2.1.35        | Chisel-MC1.12.2-0.2.1.35.jar                  | None                                     |
	| LCHIJA | thedalekmod     | (Update 49.1)            | Dalek-Mod-Update-49-1.jar                     | None                                     |
	| LCHIJA | fastleafdecay   | v14                      | FastLeafDecay-v14 (1).jar                     | None                                     |
	| LCHIJA | igauntlet       | 3.5                      | IGauntlet-Mod-1.12.2.jar                      | None                                     |
	| LCHIJA | inventorypets   | 1.5.3.1                  | inventorypets-1.12-1.5.3.1.jar                | None                                     |
	| LCHIJA | inventorytweaks | 1.63+release.109.220f184 | InventoryTweaks-1.63.jar                      | 55d2cd4f5f0961410bf7b91ef6c6bf00a766dcbe |
	| LCHIJA | ironchest       | 1.12.2-7.0.64.843        | ironchest-1.12.2-7.0.64.843.jar               | None                                     |
	| LCHIJA | jei             | 4.13.1.225               | jei_1.12.2-4.13.1.225.jar                     | None                                     |
	| LCHIJA | rope-ladder     | 1.0                      | Just-A-Rope-Ladder-Mod-1.12.2.jar             | None                                     |
	| LCHIJA | landlust        | 1.12.2-0.3.5             | Landlust-Furniture-Mod-1.12.2.jar             | None                                     |
	| LCHIJA | mantle          | 1.12-1.3.3.42            | Mantle-1.12-1.3.3.42.jar                      | None                                     |
	| LCHIJA | orespawn        | 3.3.0                    | MMD-OreSpawn-1.12.2.jar                       | d476d1b22b218a10d845928d1665d45fce301b27 |
	| LCHIJA | elevatorid      | 1.3.8                    | OpenBlocks-Elevator-Mod-1.12.2.jar            | None                                     |
	| LCHIJA | projecte        | 1.12.2-PE1.4.0           | ProjectE-1.12.2-PE1.4.0.jar                   | None                                     |
	| LCHIJA | bspkrscore      | at_0.9                   | TreeCapitator-Mod-1.12.jar                    | None                                     |
	| LCHIJA | treecapitator   | at_0.9                   | TreeCapitator-Mod-1.12.jar                    | None                                     |
	| LCHIJA | xaerominimap    | 1.16                     | Xaeros_Minimap_1.16_Forge_1.12.jar            | None                                     |
	| LCHIJA | mysticallib     | 1.12.2-1.0.0             | mysticallib-1.12.2-1.0.0.jar                  | None                                     |

	Loaded coremods (and transformers): 
Inventory Tweaks Coremod (InventoryTweaks-1.63.jar)
  invtweaks.forge.asm.ContainerTransformer
CTMCorePlugin (CTM-MC1.12.2-0.3.3.22.jar)
  team.chisel.ctm.client.asm.CTMTransformer
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 418.91' Renderer: 'GeForce GTX 1060 3GB/PCIe/SSE2'
	Launched Version: 1.12.2-forge1.12.2-14.23.5.2838
	LWJGL: 2.9.4
	OpenGL: GeForce GTX 1060 3GB/PCIe/SSE2 GL version 4.6.0 NVIDIA 418.91, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 8x Intel(R) Core(TM) i7 CPU 950 @ 3.07GHz

**Comment by Swirtzly** [comment]
```Stacktrace:
at net.minecraft.client.gui.inventory.GuiContainer.func_146977_a(GuiContainer.java:243)
at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:98)
at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:396)

-- Screen render details --
Details:
Screen name: com.swdteam.client.gui.GuiOxygenTankRefiller
Mouse location: Scaled: (221, 121). Absolute: (884, 484)
Screen size: Scaled: (442, 243). Absolute: (1768, 969). Scale factor of 4``` This is not the fault of The Tardis mod and is instead the fault of SWD Team's Dalek Mod, please contact them for support instead
